build.gradle

```diff
@@ -352,6 +352,32 @@ subprojects {
     }
   }
 
+  jacocoTestReport {
+    dependsOn test // tests are required to run before generating the report
+  }
+
+  afterEvaluate {
```

Comment:
Defining tasks in hooks slows down every gradle task. Is

Reply:
I can move this `jacocoTestCoverageVerification` task inside of the https://github.com/linkedin/venice/blob/0c90038a421f2acf6a79fab6f7b7f7f705766520/build.gradle#L302

So, it will only apply to `test` type task IIUC, but the error msg now will be printed out 3 times. Is it expected?

Reply:
Let's leave it for now, it can be gone when we don't need project specific thresholds anymore.